tests/test_wave_05.py

```diff
@@ -52,13 +52,14 @@ def test_new_genre_rec_from_empty_friends():
             }
         ]
     }
+    # Act
+    no_friends_recommendation = get_new_rec_by_genre(sonyas_data)
 
-    raise Exception("Test needs to be completed.")
-    # *********************************************************************
-    # ****** Complete the Act and Assert Portions of theis tests **********
-    # *********************************************************************
+    # Assert
+    assert len(no_friends_recommendation) == 0
+    assert not no_friends_recommendation
```

Comment:
Good job here! Essentially lines 59 and 60 test the same thing in this case. Which to keep is kind of up to you, but I think comparing the length of the `no_friends_recommendation` list to `0` is more explicit, which tends to be a better criterion.